src/frontends/tensorflow/src/op/interpolate.cpp

```diff
@@ -13,34 +13,65 @@ namespace frontend {
 namespace tensorflow {
 namespace op {
 ov::OutputVector translate_interpolate_op(const NodeContext& node) {
-    auto input = node.get_input(0);
-    auto input_sizes = node.get_input(1);
+    default_op_checks(node, 2, {"ResizeBilinear", "ResizeNearestNeighbor"});
+    auto images = node.get_input(0);
+    auto size = node.get_input(1);
+    auto op_name = node.get_name();
+    auto op_type = node.get_op_type();
 
+    // retrieve optional attribute
+    auto tf_align_corners = node.get_attribute<bool>("align_corners", false);
+    auto tf_half_pixel_centers = node.get_attribute<bool>("half_pixel_centers", false);
+
+    TENSORFLOW_OP_VALIDATION(node,
+                             !tf_half_pixel_centers || (tf_half_pixel_centers && !tf_align_corners),
```

Comment:
nit: can be simplified (!tf_half_pixel_centers || !tf_align_corners)

Reply:
I've seen it too, but it's better to leave it as is for readability